media-libs/libavif/libavif-1.2.0.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 2020-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib gnome2-utils
+
+DESCRIPTION="Library for encoding and decoding .avif files"
+HOMEPAGE="https://github.com/AOMediaCodec/libavif"
+SRC_URI="
+	https://github.com/AOMediaCodec/libavif/archive/v${PV}.tar.gz
+		-> ${P}.tar.gz
+"
+
+LICENSE="BSD-2"
+# See bug #822336 re subslot
+SLOT="0/16.1.1"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~loong ~mips ~ppc64 ~riscv ~sparc ~x86"
+IUSE="+aom dav1d examples extras gdk-pixbuf rav1e svt-av1 libyuv test"
```

Comment:
how about just naming the USE as 'yuv'? it's more intuitive which introduce YUV scaling & conversion support..

Reply:
> it's more intuitive which introduce YUV scaling & conversion support..

`libavif` already has yuv support, `libyuv` doesn't introduce new functionality, just speeds them up: https://github.com/AOMediaCodec/libavif/?tab=readme-ov-file#libyuv-dependency

Reply:
> `libavif` already has yuv support, `libyuv` doesn't introduce new functionality, just speeds them up

Ok, then be more libyuv specific, I'm fine, thanks